providers/sendinblue/src/lib/sendinblue.provider.ts

```diff
@@ -54,6 +56,75 @@ export class SendinblueEmailProvider implements IEmailProvider {
     };
   }
 
+  getMessageId(body: any | any[]): string[] {
+    if (Array.isArray(body)) {
+      return body.map((item) => item['message-id']);
+    }
+
+    return [body['message-id']];
```

Comment:
this is the same as messageId that is returned from sendMessage?

Reply:
Yes, I confirmed it by manually testing the hook. Here is the screenshot from Beeceptor where the response was sent.

![image](https://user-images.githubusercontent.com/53674742/195010585-29a9f83b-494f-4590-8cdc-b5e3238f56f3.png)